.gitignore

```diff
@@ -5,3 +5,5 @@ coverage
 .buildpath
 .project
 .settings
+/.idea
+/composer.phar
```

Comment:
not touching `.gitignore` itself, adding these things.
wanna ignore it then add it to your global ignore list